app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,375 @@
+    <string name="menu_help">Aiuto</string>
+    <string name="menu_info">Dettagli</string>
+    <string name="menu_receive">Ricevi</string>
+    <string name="menu_rename">Rinomina &#8230;</string>
+    <string name="menu_archive">Archivia</string>
+    <string name="menu_backup">Backup</string>
+
+    <string name="password_weak">Continua a scrivere &#8230;</string>
+    <string name="password_fair">Mah &#8230;</string>
+    <string name="password_good">Suvvia, puoi fare di meglio!</string>
+    <string name="password_strong">Ci siamo quasi &#8230;</string>
+    <string name="password_very_strong">Yeah baby, come un h4ck3r!</string>
+
+    <string name="label_login_wallets">Portafogli</string>
+    <string name="label_credits">Crediti</string>
+    <string name="label_ok">OK</string>
+    <string name="label_cancel">Cancella</string>
+    <string name="label_close">Chiudi</string>
+    <string name="label_wallet_advanced_details">Tocca per informazioni dettagliate</string>
+
+    <string name="label_send_success">Invio riuscito</string>
+    <string name="label_send_done">Fatto</string>
+
+    <string name="label_receive_info_gen_qr_code">Tocca per il codice QR</string>
+    <string name="info_send_prio_fees">Priorità più alta = Commissioni più alte</string>
+
+    <string name="info_xmrto_enabled">Pagamento BTC abilitato, tocca per maggiori informazioni.</string>
+
+    <string name="info_xmrto"><![CDATA[
+        <b>Hai inserito un indirizzo Bitcoin.</b><br/>
+        <i>Invierai XMR e il destinatario riceverà BTC tramite il servizio <b>XMR.TO</b>.</i>
+    ]]></string>
+
+    <string name="info_send_xmrto_success_order_label">Ordine XMR.TO</string>
+
+    <string name="info_send_xmrto_success_sending">Invio in corso</string>
+    <string name="info_send_xmrto_success_status_label">Stato:</string>
+    <string name="info_send_xmrto_success_btc">%1$s BTC</string>
+    <string name="info_send_xmrto_success_status">%1$s è %1$s (#%1$d)</string>
+
+    <string name="info_txdetails_xmrto_key">Chiave:</string>
+    <string name="info_txdetails_xmrto_address">Indirizzo:</string>
+    <string name="info_txdetails_xmrto_status">Stato:</string>
+    <string name="info_txdetails_xmrto_amount">%1$s BTC</string>
+
+
+    <string name="info_send_xmrto_paid">In attesa di conferma</string>
+    <string name="info_send_xmrto_unpaid">In attesa del pagamento</string>
+    <string name="info_send_xmrto_error">Errore XMR.TO (%1$s)</string>
+    <string name="info_send_xmrto_sent">BTC Inviati!</string>
+    <string name="info_send_xmrto_query">Richiedendo &#8230;</string>
+
+    <string name="info_send_xmrto_parms"><![CDATA[
+        <b>Puoi inviare %1$s &#8212; %2$s BTC</b>.<br/>
+        <i><b>XMR.TO</b> ti sta <u>attualmente</u> concedendo un tasso di cambio di <b>%3$s BTC</b></i>.
+    ]]></string>
+    <string name="info_send_xmrto_zeroconf"><![CDATA[
+        <i>Un ammontare fino a <b>%1$s BTC</b> sarà inviato <u>istantaneamente</u>!</i>
```

Comment:
You could remove `Un` here. Let's trim the text when possible

Reply:
Agreed. I shrank it as much as possible.